core/src/wallet/algorand/AlgorandAccount.cpp

```diff
@@ -94,26 +96,36 @@ namespace algorand {
 
     bool Account::putBlock(soci::session& sql, const api::Block& block)
     {
-        if (BlockDatabaseHelper::putBlock(sql, block)) {
-            emitNewBlockEvent(block);
-            return true;
-        }
-        return false;
+        emitNewBlockEvent(block);
```

Comment:
I assume that we don't have to return a bool for this method? void is enough?
because we don't use the return value.
Also, in the old code we emit a Block Event when it is a new block (the block doesn't exist in the DB)
in the new code, we always emit this event. 
Does it change the logic?